### DISCUSSION TITLE
Does RefindPlus Enable Running Older OSX in Newer Macs?

### DISCUSSION THREAD

**Discussion by juanpc2018** [dakanji/RefindPlus#63] [General]
everybody wants the opposite,
Run a New OSX in Older Macs,

but owners of legal software & hardware, that invested $$$ back in the day,
want to run the old Software & Hardware "like scanners" in a Faster Mac
want to buy a New Mac HW, 
but don´t want to pay for the software again, 
some manufacturers discontinued the Old SW & HW, 
there is No way, No New SW for the Old HW, can´t be purchased,
other developers require to buy a completely New Hardware to use the New Software in a New Mac, 
New SW does Not work with older HW, feature was removed.
etc...
many reasons.
usually most desired OSX to run are OSX Mavericks 10.9.5 and OSX SnowLeopard 10.6.8
in Newer Macs.
OSX Mavericks 10.9.5 latest update was in year 2016
but OSX Mavericks does Not work in Mac Mini 2014.
https://en.wikipedia.org/wiki/Mac_Mini#Supported_operating_systems

OSX SnowLeopard has Rosetta v1 that allows to run 32-Bit PowerPC G4 G5 software,
mostly abandonware but available in websites like MacGarden and MacRepository.

i´ve tested Clover, and rEFInd standard, None works,
OSX Snow has the Apple logo but stops loading from USB2.0 drive, 
OSX Mavericks gives a forbiden icon.
= it´s impossible to run Older Mac OSX on a Genuine Mac.
but is possible to run OSX in a Non Genuine Mac.
that´s insane.
someone has lost the mind.

some people install Parallels or VMWave Fusion Player 12.x
but...
is Not the same as True Native,
from the test i´ve done Parallels feels better than VMWware FusionPlayer 12.x 
but 1 time only payment license is limited to 4-Threads and the HDD/SSD is also limited to 60 MB/s.
it´s a paid Downgrade. from 800MB/s M.2 drive to 60MB/s, makes No sense.
No logic to buy a faster machine a Parallels license, to use it slower than an older machine.

Fusion Player 12.x it´s free, and Not limited, but...
both have same problem:
there is No FireWire Pass Through Virtual Driver.
some older apps require FireWire, but New Mac does Not have FireWire,
the only Option is the Thunderbol to FireWire adapter, but as far as i know does Not have drivers for older OSX.
maybe looking really hard in the ASIX website, or chip manufacturer.
similar happens with Apple USB Ethernet adapter, does have drivers for older OSX and works ok when installed.

installing rEFInd in Kubuntu was very easy because it has a PPA

> $ sudo apt-add-repository ppa:rodsmith/refind
> $ sudo apt-get update
> $ sudo apt-get install refind

installing Clover in Kubuntu to a USB was also easy because it has a CD image,
using Disks from gnome-disk-utility 
and restoring the D Image to the USB drive, works ok

but RefindPlus seems more complicated...
to rename & replace the .efi and .config files seems the better option.
but requires booting from a different OS that allows to see & edit the partition.

**Comment by dakanji** [comment]
RefindPlus/rEFInd do not control running anything.

They are simply a way to start bootloaders. Bootloaders are files that fire up operating systems. The process of firing up an operating system is called "booting".

If your computer is able to run an operating system, then starting a bootloader will result in that operating system being booted. If it can't, booting will fail. 

RefindPlus/rEFInd have nothing to do with the capabilities of your computer. They are boot managers ... they allow users to manage bootloaders.
